examples/solvers/pdhg_denoising_L1_HuberTV.py

```diff
@@ -0,0 +1,123 @@
+"""Total variation denoising using PDHG.
+
+Solve the L1-HuberTV problem
+
+        min_{x >= 0} ||x - d||_1 + lam TV_gamma(x)
+
+where ``grad`` the spatial gradient and ``d`` is given noisy data.
```

Comment:
there is no "grad" above, but i personally like the wrong version which is `||grad(x)||_1`

Reply:
True, modified it similar to your suggestion.